subsys/dm/dm.c

```diff
@@ -207,6 +207,9 @@ static mpsl_timeslot_signal_return_param_t *mpsl_timeslot_callback(
 		dm_api_call = TIMESLOT_RESCHEDULE;
 		k_msgq_put(&dm_api_msgq, &dm_api_call, K_NO_WAIT);
 		break;
+	case MPSL_TIMESLOT_SIGNAL_OVERSTAYED:
+		LOG_ERR("overstayed the timeslot: consider increasing DM_TIMESLOT_OVERHEAD_US");
+		break;
 	default:
 		break;
```

Comment:
Here we should consider printing an error message when it gets an unexpected signal